### ISSUE TITLE
Get beta-smoothed model confidence

### ISSUE THREAD

**Issue by Mahhos** [ZhiluZhang123/neurips_2020_distillation#1]
Hi. Thanks for the great repo. I want to use beta smoothing in a text classification task and plan to calculate ECE using beta smoothing. First, I need to get model softmaxes using beta smoothing (in order to calculate ECE). 

I guess i can return `beta_label` from `soft_beta_loss` function as beta-smoothed model confidence. My question is what is `outputs_orig` in the `soft_beta_loss(outputs, labels, beta, outputs_orig, num_classes=10):`? `outpus` should be model logits/probabilities, `labels` should be ground truth labels, how about `outputs_orig`?
